test/db/cmd/cmd_i

```diff
@@ -386,8 +296,10 @@ iA
 iAj
 EOF
 EXPECT=<<EOF
-0   0x00000000 266108 mips_32 mips1 o32 
-{"bins":[{"arch":"mips","bits":32,"offset":0,"size":266108,"isa":"mips1","features":"o32","machine":"MIPS R3000"}]}
+num offset     size   arch              machine    
```

Comment:
For example, you kept the number here.

Reply:
oh, good catch i should remove this. I was also printing `0` always...